File: cassandra/src/main/java/com/spdeveloper/cassandra/controller/PersonController.java
package com.spdeveloper.cassandra.controller;

import com.spdeveloper.cassandra.service.PersonsService;
import com.spdeveloper.cassandra.service.dto.PersonDTO;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.*;

@RestController
@RequestMapping("/persons")
public class PersonController {
    private final PersonsService personsService;

    public PersonController(PersonsService personsService) {
        this.personsService = personsService;
    }
    @PostMapping
    public ResponseEntity<?> insert (@RequestBody PersonDTO dto){
        return ResponseEntity.ok(personsService.insert(dto).getId());
    }
    @GetMapping
    public ResponseEntity<?> findAll (){
        return ResponseEntity.ok(personsService.findAll());
    }
}

File: kotlin/first-app-kptolin/src/main/kotlin/com/omidmohebbise/firstappkt/firstappkotlin/controller/PersonController.kt
package com.omidmohebbise.firstappkt.firstappkotlin.controller

import com.omidmohebbise.firstappkt.firstappkotlin.entity.PersonEO
import com.omidmohebbise.firstappkt.firstappkotlin.repository.PersonJpaRepository
import org.springframework.http.HttpStatus
import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.CrossOrigin
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.PathVariable
import org.springframework.web.bind.annotation.RequestMapping


@Controller
@CrossOrigin
@RequestMapping("/persons")
class PersonController (private val repository: PersonJpaRepository){

    @GetMapping("/{id}")
    fun getUserById(@PathVariable("id") userId: Long): ResponseEntity<PersonEO>{
        return ResponseEntity(repository.findById(userId).orElse(null) ,HttpStatus.OK);
    }
}